eos/models/wilson-coefficients.hh

```diff
@@ -168,23 +168,37 @@ namespace eos
          * These two operators are defined via
          *
          * H^eff = 4 G_F / sqrt(2) V_tb V_ts^* (
-         *             C_L alpha_e / (2 pi) * [sbar gamma^mu P_L b] [nubar gamma_mu P_L nu]
-         *             C_R alpha_e / (2 pi) * [sbar gamma^mu P_R b] [nubar gamma_mu P_L nu]
+         *             C_L alpha_e / (2 pi) * [sbar gamma^mu P_L b][nubar gamma_mu P_L nu] 
+         *             C_R alpha_e / (2 pi) * [sbar gamma^mu P_R b][nubar gamma_mu P_L nu] 
          *         )
-         *       = 4 G_F / sqrt(2) V_tb V_ts^* ( C_L O_L + C_R O_R )
+         *       = 4 G_F / sqrt(2) V_tb V_ts^* ( C_VLL O_VLL + C_VRL O_VRL )
+         * 
+         * Furthermore, we can add the scalar operators [FLS:2021A], eq. (2)
+         * O_SLL = [sbar P_L b][bar(nu_L^c) P_L nu], where nu_L^c = C nubar_L^T, C = i gamma^2 gamma^0
+         * O_SRL = [sbar P_R b][bar(nu_L^c) P_L nu], where nu_L^c = C nubar_L^T, C = i gamma^2 gamma^0
+         * 
+         * And tensor operators [FLS:2021A], eq. (2)
+         * O_TLL = [sbar sigma_{mu nu} P_L b][bar(nu_L^c) sigma^{mu nu} P_L nu], where nu_L^c = C nubar_L^T, C = i gamma^2 gamma^0
+         * 
+         * 
          *
          * C++ idx -> operator
-         * 0       -> O_L
-         * 1       -> O_R
+         * 0       -> O_VLL
```

Comment:
Your comment differes from the names used for the class, i.e., you write ``O_VLL`` but the WC is named ``cVL``.
If given the choice, I would prefer the latter convention.